modules/amf-registration/amf-registration-service/service.xml

```diff
@@ -24,24 +24,27 @@
 
 		<!-- Other fields -->
 
-		<column name="field1" type="String" />
-		<column name="field2" type="boolean" />
-		<column name="field3" type="int" />
-		<column name="field4" type="Date" />
-		<column name="field5" type="String" />
+		<column name="first_name" type="String" />
+		<column name="last_name" type="String" />
+		<column name="email_address" type="String" />
+		<column name="username" type="String" />
+		<column name="gender" type="String" />
+		<column name="birthday" type="Date" />
+		<column name="password" type="String" />
+		<column name="home_phone" type="int" />
+		<column name="mobile_phone" type="int" />
+		<column name="address" type="String" />
+		<column name="address2" type="String" />
+		<column name="city" type="String" />
+		<column name="state" type="String" />
+		<column name="zip" type="int" />
```

Comment:
use camelcase for column names: for instance, use `firstName` instead of `first_name`

Reply:
use `accountName` to avoid conflits